compiler/dyno/lib/resolution/scope-queries.cpp

```diff
@@ -369,7 +376,7 @@ static bool doLookupInImports(Context* context,
     }
   }
 
-  if (scope->autoUsesModules()) {
+  if (scope->autoUsesModules() && !skipPrivateVisibilities) {
```

Comment:
Is auto-use private, too? If not, what's the reason for including `skipPrivateVisibilities` here?

Reply:
Yes, the auto-use is private. The `!skipPrivateVisibilities` check is included here since the auto-`use`d module(s?) are not included in the visibility clauses iterated through above.